cmd/sign.go

```diff
@@ -94,7 +95,17 @@ var signCmd = &cobra.Command{
 			panic("failed to parse certificate: " + err.Error())
 		}
 		fmt.Printf("Received signing Cerificate: %+v\n", cert.Subject)
-		},
+
+		mimetype, err := utils.GetFileType(viper.GetString("artifact"))
+		if err != nil {
+			fmt.Println(err)
```

Comment:
we should probably return or panic here instead of continuing? same as on 106

Reply:
Good point, I will sys exit with an error exit code (as its a CLI), sound good @dlorenc ?